src/cmps/button/Button.tsx

```diff
@@ -1,26 +1,31 @@
-//Button component
-
-import React, { MouseEvent } from "react";
+import React, { MouseEvent } from 'react';
 import './Button.css';
 
 // without state use react stateless function
 type props = {
     isDisabled?: boolean;
     value: string;
     onClick(e: MouseEvent<HTMLElement>): void;
-}
+};
 
 const Button: React.FC<props> = ({ isDisabled, value, onClick: handleClick }) => {
```

Comment:
I can't find a reason for renaming `onClick` with `handleClick`

Reply:
OK